### PR TITLE
Code Quality: Refactored the settings controls

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

As the title.

### Motivation and Context

To improve performance of Settings dialog.

- Little UI change
- Large codebase change

### Performance Improvements

_None yet_

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app
   2. Click settings button

### Screenshots

_None yet_

### CODE REVIEW

**Comment by yaira2** [comment]
> Little UI change

Is this still the case?

**Comment by 0x5bfa** [comment]
Expander UI. Looks like Windows 11 Settings Expander style.

File: src/Files.App/Views/Settings/AboutPage.xaml [hunk 1]
```diff
@@ -3,44 +3,36 @@
...
-			HorizontalAlignment="Stretch"
-			VerticalAlignment="Stretch"
-			Spacing="4">
+	<Grid x:Name="RootGrid">
+		<StackPanel Spacing="{StaticResource SettingsCardSpacing}">
 			<StackPanel.ChildrenTransitions>
-				<TransitionCollection>
-					<RepositionThemeTransition />
-				</TransitionCollection>
+				<EntranceThemeTransition FromVerticalOffset="50" />
```
Comment (yaira2) [inline src/Files.App/Views/Settings/AboutPage.xaml:21]: Can you split the unrelated changes into a separate PR?

Comment (yaira2) [reply src/Files.App/Views/Settings/AboutPage.xaml:21]: As a general rule of thumb, we try to discuss requirements before starting on a PR. This way we can set the scope and approve any changes before making them.

Comment (0x5bfa) [reply src/Files.App/Views/Settings/AboutPage.xaml:21]: Done.

**Comment by 0x5bfa** [comment]
Can you review @yair100 again

**Comment by yaira2** [comment]
Before continuing with this, we need to remove the old settings controls. We'll have to update the properties pages as well to make sure they are all using the new control.
